### ISSUE TITLE
Can't I use test_ Bread_NFM. py for testing after training with train_CAN.py?

### ISSUE THREAD

**Issue by sisisia** [mingcv/Bread#10]
train_CAN.py ：colors = self.model_canet(torch.cat([texture_in, cb_in, cr_in, texture_gt, cb_en, cr_en], dim=1))
test_Bread_NFM.py :   colors = self.model_canet( torch.cat([texture_in, cb_in, cr_in, texture_ns], dim=1))
Their inputs are different.

**Comment by mingcv** [comment]
[train_MECAN.py](https://github.com/mingcv/Bread/blob/main/train_MECAN.py) is the final decision for the color adaption network.

**Comment by mingcv** [comment]
It seems that there is no further question, we'll go ahead and close this issue for now. If you have additional questions, feel free to open a new issue.
